.github/workflows/e2e.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: e2e
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
humm, taking a look at it again, this will not work when running for fork PRs, we need to use `pull_request_target` for that, but that creates a possible security issue.
So we might use a test key or dont run in PRs

Reply:
see https://github.com/kubernetes-sigs/release-sdk/runs/7914289424?check_suite_focus=true